devicetypes/smartthings/zigbee-metering-dimmer.src/zigbee-metering-dimmer.groovy

```diff
@@ -0,0 +1,133 @@
+/**
+ *  Copyright 2021 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import physicalgraph.zigbee.zcl.DataType
+metadata {
+	definition (name: "ZigBee Metering Dimmer", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light",  mnmn: "SmartThings", vid:"generic-dimmer-power-energy") {
+
+		capability "Actuator"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Power Meter"
+		capability "Energy Meter"
+		capability "Switch"
+		capability "Switch Level"
+		capability "Health Check"
+
+		// Enbrighten/Jasco
+		fingerprint manufacturer: "Jasco Products", model: "43082", deviceJoinName: "Enbrighten Switch" //Enbrighten, in-Wall Smart Dimmer With Energy Monitoring 43082, Raw Description: 01 0104 0101 00 08 0000 0003 0004 0005 0006 0008 0702 0B05 02 000A 0019
```

Comment:
I believe we use "Light" for dimmers, right @PKacprowiczS ?

Reply:
@greens Isn't this capability deprecated ?

Reply:
I meant the join name, not the capability. Instead of "Enbrighten Switch" I believe the convention would be "Enbrighten Light".

Reply:
I think "Light" is used in bulb fingerprints.

Reply:
Might need to say "Dimmer Switch" then, going off of zigbee-dimmer.groovy

Reply:
yup, You are right.